README.md

```diff
@@ -21,8 +21,12 @@ Google Cloud Platform products.
 
 ## Supported Platforms
 
-The Cloud Code for IntelliJ plugin supports JetBrains IDEs version 2018.2+: IntelliJ IDEA Ultimate 
-or Community, PyCharm Professional or Community, Webstorm, Goland, and other JetBrains IDEs.
+The Cloud Code for IntelliJ plugin supports JetBrains IDEs version 2018.2+: 
+* [IntelliJ IDEA Ultimate or Community](https://www.jetbrains.com/idea/)
+* [PyCharm Professional or Community](https://www.jetbrains.com/pycharm/)
+* [Webstorm](https://www.jetbrains.com/webstorm/)
```

Comment:
Also, “WebStorm”

Reply:
lol, good we figured that out with this change! Thanks.